File: Iniciante/1072 - Intervalo 2/Python/1072.py
def main():
    INTERVAL_START = 10
    INTERVAL_END = 20
    N = int(input())
    values_in = 0
    values_out = 0
    for _ in range(N):
        value = int(input())
        if INTERVAL_START <= value <= INTERVAL_END:
            values_in += 1
        else:
            values_out += 1
    print(f"{values_in} in")
    print(f"{values_out} out")


if __name__ == "__main__":
    main()

File: Iniciante/1072 - Intervalo 2/JavaScript/1072.js
function main() {
  const input = require("fs").readFileSync("/dev/stdin", "utf8");
  const lines = input.split("\n");
  const INTERVAL_START = 10;
  const INTERVAL_END = 20;
  const N = parseInt(lines.shift());
  let valuesIn = 0;
  let valuesOut = 0;
  for (i = 0; i < N; i++) {
    let value = parseInt(lines.shift());
    if (value >= INTERVAL_START && value <= INTERVAL_END) {
      valuesIn++;
    } else {
      valuesOut++;
    }
  }
  console.log(`${valuesIn} in`);
  console.log(`${valuesOut} out`);
}

main();
